jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -625,6 +658,23 @@ public MethodHandle findStatic(Class<?> clazz, String methodName, MethodType typ
 			return handle;
 		}
 		
+		/*[IF Sidecar19-SE]*/
+		/**
+		 * Return a MethodHandle to a virtual method.  The method will be looked up in the first argument 
+		 * (aka receiver) prior to dispatch.  The type of the MethodHandle will be that of the method
+		 * with the receiver type prepended.
+		 * 
+		 * @param clazz - the class defining the method
+		 * @param methodName - the name of the method
+		 * @param type - the type of the method
+		 * @return a MethodHandle that will do virtual dispatch on the first argument
+		 * @throws IllegalAccessException - if method is static or access is refused or clazz's package is not exported to the current module 
+		 * @throws IllegalAccessError - if the type's packages are not exported to the current module 
```

Comment:
Has the RI thrown IA**Error** from findVirtual?

The convention is usually that the VM throws IllegalAccessError and reflective style operations throw Exceptions.  We're using the same code to do double duty as the VM is calling into Java for these lookups.

I would expect exception / error conversion to happen at the boundaries.

Reply:
> Has the RI thrown IAError from findVirtual?

Yes.  This matches the behaviour of the reference implementation.